etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
firecfg?

Reply:
No, paste error in github editor

Reply:
I meant should it be added to firecfg? 

Could have been clearer here but if you ask the same question for every second new profile you become lazy.

Reply:
I dont see a reason not to. It works perfectly.